.github/workflows/ci_test.yml

```diff
@@ -14,7 +14,33 @@ env:
   TOXARGS: '-v'
 
 jobs:
+
+  check_commit:
```

Comment:
I think some comments here to describe what this is actually for would be helpful — why are we doing this?